changelog/@unreleased/pr-1051.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: Undertow endpoints disallow request body collections with null values
+    when `--nonNullCollections` is set
```

Comment:
Need to update this changelog

Reply:
I don't really want anyone except you to know about the new flag.